PerfectNumberTest.java

```diff
@@ -16,5 +17,11 @@ public void should_return_true_As_1_is_factorOf_6() {
         PerfectNumber perfectNumber = new PerfectNumber();
         assertTrue(perfectNumber.isFactor(1,6));
     }
+    
+    @Test
+    public void should_return_false_As_4_is_not_factorOf_6() {
+        PerfectNumber perfectNumber = new PerfectNumber();
+        assertFalse(perfectNumber.isFactor(4,6));
+    }
 

```

Comment:
Significance of the test case should_return_false_As_4_is_not_factorOf_6 in finding the Perfect Number?